tests/unit/amo/components/TestGetFirefoxBanner.js

```diff
@@ -29,6 +29,11 @@ import {
   shallowUntilTarget,
   userAgents,
 } from 'tests/unit/helpers';
+// eslint-disable-next-line no-unused-vars
+import tracking from 'amo/tracking';
```

Comment:
Is this required to make `jest.mock` work?

Reply:
Yes, exactly.

Reply:
Actually, we don't need it. I never tested without it, and just did, and I guess it's not needed at all. I'll remove these imports and comments.